cmd/brew-postgresql-upgrade-database.rb

```diff
@@ -65,7 +65,7 @@
   initdb_run = true
 
   (var/"log").cd do
-    system "#{bin}/pg_upgrade",
+    raise unless system "#{bin}/pg_upgrade",
```

Comment:
Could do `safe_system` alternatively? I'm happy with either.

Reply:
I did not know about this utils, it seems perfectly fitted for the job.